pkg/sql/colexec/table_function/fulltext_tokenize.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package table_function
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/matrixorigin/matrixone/pkg/common/fulltext"
+	"github.com/matrixorigin/matrixone/pkg/common/moerr"
+	"github.com/matrixorigin/matrixone/pkg/container/batch"
+	"github.com/matrixorigin/matrixone/pkg/container/bytejson"
+	"github.com/matrixorigin/matrixone/pkg/container/types"
+	"github.com/matrixorigin/matrixone/pkg/container/vector"
+	"github.com/matrixorigin/matrixone/pkg/sql/colexec"
+	"github.com/matrixorigin/matrixone/pkg/vm"
+	"github.com/matrixorigin/matrixone/pkg/vm/process"
+	"github.com/matrixorigin/monlp/tokenizer"
+)
+
+type FullTextEntry struct {
+	DocId any
```

Comment:
Replay with
```golang
type FullTextEntry struct {
     DocId []byte
     Type types.T
     .....
}
```
`GetAny` and `AppendAny` are very convenient, but have side effects, it is recommended to use them only in some TEST scenarios, it is best to delete these two

Reply:
comment added for GetAny and AppendAny to warn that memory will escape to heap for any type that result in slow GC.